sunpy/net/helioviewer.py

```diff
@@ -189,6 +192,63 @@ def download_jp2(self, date, observatory=None, instrument=None, detector=None,
 
         return self._get_file(params, directory=directory, overwrite=overwrite)
 
+    def get_jp2_header(self, date, observatory=None, instrument=None, detector=None,
+                       measurement=None, jp2_id=None, directory=None, overwrite=False):
```

Comment:
Why are `directory=None, overwrite=False` here?

Reply:
I wanted to ask this. Do we wish to provide the freedom to download the headers in any of the directory or shall we remove this?

I mean do we really need to keep these two parameters or can we omit them?

Reply:
We are returning a dictionary. I don't think we are meant to download anything.

Reply:
But to convert the XML data into dictionary I first download the XML file and then convert it into a dictionary. That is also the reason for using `requests` module

Reply:
Do you have to create a downloaded file to read it and parse it?

Reply:
I guess so. That is an easy way of converting the XML file into a dictionary

Reply:
Can't you convert the response directly?

Reply:
I don't think so because the API just redirects you to a page containing the XML response.